engine/render/r_main.cpp

```diff
@@ -1724,3 +1893,83 @@ void r_renderer_c::DoScreenshot(image_c* i, const char* ext)
 		sys->con->Print(fmt::format("Wrote screenshot to {}\n", ssname).c_str());
 	}
 }
+
+r_renderer_c::RenderTarget& r_renderer_c::GetDrawRenderTarget()
+{
+	return rttMain[1 - presentRtt];
+}
+
+r_renderer_c::RenderTarget& r_renderer_c::GetPresentRenderTarget()
+{
+	return rttMain[presentRtt];
+}
+
+// ============================================
+// MurmurHash implementation from public domain

```

Comment:
Mind sourcing this?  Seeing differences between this and (for example) https://chromium.googlesource.com/external/smhasher/+/58dd8869da8c95f5c26ec70a6cdd243a7647c8fc/MurmurHash3.cpp

Reply:
I'll try to hunt down where it's from as it's wandered across my projects over time.

Reply:
Origin now referenced via link in code comment.